client/src/components/dashboard/AdminDashboard.js

```diff
@@ -27,6 +27,9 @@ const AdminDashboard = ({
           <Col>
             <Button href="/managepets">Manage Pet Profiles</Button>
           </Col>
+          <Col>
+            <Button href="/admineditprofile">View/Update Admin Profile</Button>
```

Comment:
If we are just updating the password, how about change the text to "Change Password"? We can keep the href as it is for future usage, but for now, I feel like "Change Password" is less confusing for a user.

Reply:
we only have password to update for now, but profile could have more and password is part of profile. since it is part of dashboard operations, it feels strange to say only "Change Password" on dashboard?